packages/flutter_tools/test/general.shard/web/devfs_web_test.dart

```diff
@@ -942,12 +942,12 @@ void main() {
       useSseForDebugBackend: true,
       nullAssertions: true,
       buildInfo: const BuildInfo(
-          BuildMode.debug,
-          '',
-          treeShakeIcons: false,
-          dartDefines: <String>[
-            'FLUTTER_WEB_AUTO_DETECT=true',
-          ]
+        BuildMode.debug,
+        '',
+        treeShakeIcons: false,
+        dartDefines: <String>[
+          'FLUTTER_WEB_AUTO_DETECT=true',
+        ]
```

Comment:
I only see whitespace changes. How would the test prevent this regression in the future?

Reply:
This won't. I have it fixed because I saw the inconsistency.